CHANGES.md

```diff
@@ -16,6 +16,14 @@ To be released.
  -  (Libplanet.Net) Changed the return type of
     `BlockCandidateTable<T>.GetCurrentRoundCandidate()` from `List<Block<T>>?`
     to `Branch<T>?`.  [[#2822]]
+ -  Added a `logs` parameter to the below methods in `IActionRenderer<T>`
+    interface and its implementations.  [[#2826]]
```

Comment:
We should add markdown link like the below:

```markdown
[#2826]: https://github.com/planetarium/libplanet/2826
```

Reply:
Amended. aa67d5e7c